package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npx hardhat test"
+    "test-configuration-tests": "npx hardhat test --dirname=configuration-tests",
```

Comment:
Would it be better to have these two scripts in `starknet-hardhat-plugin`? And here in the example repo have just the `test` script? This is probably related to the target branch choice.